package.json

```
@@ -5,15 +5,17 @@
   "module": "dist/webexSDKComponentAdapter.esm.js",
   "scripts": {
     "build": "rollup -c",
-    "linter": "eslint src/",
+    "linter": "eslint . --ignore-path .gitignore",
     "release": "semantic-release",
     "release:debug": "semantic-release --debug",
     "release:dry-run": "semantic-release --dry-run",
     "start": "nodemon -r dotenv/config scripts/start.js",
     "serve": "parcel scripts/index.html",
-    "test": "jest",
-    "test:integration": "jest \"^(?=.*integration.test.js).*\"",
-    "test:unit": "jest \"^(?=.*.test.js)(?!.*integration).*\"",
+    "jest": "jest",
+    "cypress": "cypress open",
+    "test:integration": "cypress run --browser chrome",
+    "test:integration:ci": "npm run test:integration -- --reporter junit --reporter-options \"mochaFile=test_results/cypress/results-[hash].xml,toConsole=true\"",
+    "test:unit": "jest",
```

```suggestion
    "test:unit": "jest --silent",
```
this should be silent by default to be honest to hide all the console.errors from tests that are testing for errors